docs/UserGuide.md

```diff
@@ -97,46 +97,50 @@ Expressions without `...` at the end takes the last parameter as the argument wh
 ### Adding a tag with filepath : `tag`
 
 Tags a file with a unique tag name.<br>
-You can add a nickname to a file. Optionally, you can add some labels to the tag for categorisation.
-The `FILE_PATH` field can accept both absolute and relative file path from your current directory in HelloFile.
+You can add a nickname to a file. This make it easier to manage only some specific files that you want such as frequently used files, etc.
+
+Please take note of the following:
+* You can add any number of labels to the tag (including 0).
+* The `FILE_PATH` field can accept both absolute and relative file path from your current directory specified in the internal File Explorer of HelloFile.
```

Comment:
maybe which was specified is better, just to avoid words confusion

Reply:
Changed!